lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -63,9 +63,8 @@ class DetailsRenderer {
         return null;
 
       default: {
-        // @ts-ignore tsc thinks this unreachable, but ts-ignore for error message just in case.
-        const detailsType = details.type;
-        throw new Error(`Unknown type: ${detailsType}`);
+        // @ts-ignore tsc thinks this unreachable, but ts-ignore so we can render an unknown element.
```

Comment:
Maybe mention that we want to be moderately forward compatible?

```suggestion
// Tsc thinks this is unreachable. But renderers should be forward compatible with new unexpected detail types.
// @ts-ignore
```